examples/dt/nova/nova-three-cells/README.md

```diff
@@ -0,0 +1,185 @@
+# Deployed Topology - Nova/MultipleCells
+
+Deploys an environment with three cells instead of the standard two cells
+
+## Purpose
+
+Focused on leveraging multiple nodesets to deploy two distinct cells 
+(cell1 and cell2) in addition to cell0. Computes will be allocated to both 
+cells
```

Comment:
Would be nice to describe the mapping. Which compute will be connected to which cell.

Reply:
Ack